src/alert.scss

```diff
@@ -21,16 +21,19 @@ $block: #{$fd-namespace}-alert;
   @mixin fd-alert-icon-container {
     position: absolute;
     width: 2.5rem;
-    top: 0.625rem;
+    top: 0.4375rem;
     left: 0;
   }
 
   @mixin fd-alert-close-btn-container {
     position: absolute;
-    width: 2rem;
-    height: 1.625rem;
-    top: 0.375rem;
+    top: 0.125rem;
     right: 0.125rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 2rem;
+    height: 1.625rem;
```

Comment:
The interaction states of the button should follow these of the Transparent button